anti-analysis/anti-vm/vm-detection/check-for-windows-sandbox-via-subdirectory.yml

```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: check for windows sandbox via subdirectory
+    namespace: anti-analysis/anti-vm/vm-detection
+    authors:
+      - "echernofsky@google.com"
+    scope: basic block
+    att&ck:
+      - Defense Evasion::Virtualization/Sandbox Evasion::System Checks [T1497.001]
+    mbc:
+      - Anti-Behavioral Analysis::Virtual Machine Detection [B0009]
+    references:
+      - https://www.elastic.co/security-labs/qbot-malware-analysis
+    examples:
+      - ffaf9cd085434fd5230511ab895ba494:0x10015A6A
```

Comment:
if this is the according testfile, let's change to use sha256 sum here
https://github.com/mandiant/capa-testfiles/pull/166